console/src/main/java/org/georchestra/console/events/RabbitmqEventsAutoConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package org.georchestra.console.events;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnExpression;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.*;
+import org.springframework.context.support.ClassPathXmlApplicationContext;
+
+import javax.annotation.PostConstruct;
+
+@Configuration
+@ImportResource({ "file:webapps/console/WEB-INF/spring/rabbit-listener-context.xml",
+        "file:webapps/console/WEB-INF/spring/rabbit-sender-context.xml" })
```

Comment:
Same remark as https://github.com/georchestra/georchestra/pull/4037#issuecomment-1728247260 : I would move the xml files into the classpath and resolve them using the classpath:// scheme, because there is no guarantee that the servlet container will run in a specific directory where a `webapps/` subdirectory will be available.